container/lxd/initialisation_test.go

```diff
@@ -327,7 +333,7 @@ func (s *InitialiserSuite) TestFindAvailableSubnetUpperBoundAndLowerBoundInUse(c
 	})
 	subnet, err := findNextAvailableIPv4Subnet()
 	c.Assert(err, gc.IsNil)
-	c.Assert(subnet, gc.Equals, "1")
+	c.Assert(subnet, gc.Equals, "4")
 }
```

Comment:
I am not sure how juju would know of these other subnets though.
Randomizing 2nd octet too is possible, just need to change a bit the code asking for a subnet to ask for a broader range.